### DISCUSSION TITLE
Directory-specific Theme?

### DISCUSSION THREAD

**Discussion by superMDguy** [tailwindlabs/tailwindcss#9302] [Help] [answered]
This is kind of a long shot, but I'm working on a project where it'd be great to have a different theme applied only to a specific directory. Any ideas on a direction that might work with this? Looked into plugins but I'm not sure if that would work for this use-case.

**Comment by wongjn** [comment, marked as answer]
If it's only colors, you can try using [CSS variables](https://tailwindcss.com/docs/customizing-colors#using-css-variables), overriding them for the directory. How you actually write the CSS variables into your HTML depends on your stack. For customizations other than colors, you could look at adding a separate `input.css` file for the directory, then converting your `postcss.config.js` to [export a function](https://github.com/postcss/postcss-load-config#context) so that you can then change your Tailwind config (either inline or specifying a different tailwind config file) depending on the input CSS file.
